### ISSUE TITLE
Get R-libraries downloading properly when testing in devtools

### ISSUE THREAD

**Issue by gltrost** [gltrost/Microbiomics#3]
Currently, not all r files all tested because these files have dependecies that won't download properly with github devtools.

**Comment by gltrost** [comment]
Attempted to add the lines to r-tests.yml : 
```
    - run: Rscript -e 'install.packages(c("tidyr","dplyr","tidyverse","hrbrthemes","tm","proustr","VennDiagram","RColorBrewer","data.table","dplyr",
"formattable","tidyr"),dependencies=TRUE,repos="http://cran.rstudio.com/")'
```
and 
```
    - run: Rscript Code/utils/venn-diagram.r

```

I ended up with the following erros seen at [Address issue #3 by adding test cases for all r files. R Test 1 #31
](https://github.com/gltrost/Microbiomics/actions/runs/441129910): 

```
Run Rscript Code/utils/venn-diagram.r
Error: Error in library(tidyverse) : there is no package called ‘tidyverse’
Execution halted
Error: Process completed with exit code 1.
```

Before that error, there were some other errors when trying to run `install.packages(<packages>)`: 
```
configure: error: cannot determine poppler-glib compile/link flags
ERROR: configuration failed for package ‘Rpoppler’
```

```
ERROR: configuration failed for package ‘curl’
```

```
ERROR: dependency ‘curl’ is not available for package ‘httr’
```

```
ERROR: dependencies ‘httr’, ‘rvest’ are not available for package ‘tidyverse’

```

to list a few. This is what leads to the `Error: Error in library(tidyverse) : there is no package called ‘tidyverse’
`-error.

**Comment by gltrost** [comment]
Adding the line `- run: sudo apt-get install libcurl4-openssl-dev` to `r-tests.yml` fixed the above issue! It adds `curl` and other packages that were needed. 
The next error to show up is 
```
Loading required package: grid
Loading required package: futile.logger
Error: Error in setwd("~/Microbiomics/Code") : cannot change working directory
Execution halted
Error: Process completed with exit code 1.
```

The line `setwd("~/Microbiomics/Code")` is used for the next lines 

```
genecards <- read.csv('/data/genecards_celiac_and_duodenum.csv',TRUE)
ctd <- read.csv("/data/ctd_celiac_simple.csv",TRUE)
ncbi <- read.csv("/data/ncbi_celiac.csv",TRUE)
disgenet <- read.csv("/data/disgenet_celiac_table.csv",TRUE)
```

but I can maybe change these to 

```
genecards <- read.csv('~/Microbiomics/Code/data/genecards_celiac_and_duodenum.csv',TRUE)
ctd <- read.csv("~/Microbiomics/Code/data/ctd_celiac_simple.csv",TRUE)
ncbi <- read.csv("~/Microbiomics/Code/data/ncbi_celiac.csv",TRUE)
disgenet <- read.csv("~/Microbiomics/Code/data/disgenet_celiac_table.csv",TRUE)

```

**Comment by gltrost** [comment]
After doing the above changes of getting rid of the `setwd` line, I now get 

```
Loading required package: grid
Loading required package: futile.logger
Error: Error in file(file, "rt") : cannot open the connection
Calls: read.csv -> read.table -> file
In addition: Warning message:
In file(file, "rt") :
  cannot open file '/home/runner/Microbiomics/Code/data/genecards_celiac_and_duodenum.csv': No such file or directory
Execution halted
Error: Process completed with exit code 1.
```

**Comment by gltrost** [comment]
Issue resolved with #5 . A new issue should be opened to handle automatic testing.
